api/controllers/project.js

```diff
@@ -522,11 +532,12 @@ handleGetPins = async (projectId, roles, sortBy, pageSize, pageNum, username, re
         Utils.recordAction('Get', 'Pin', username, projectId && projectId.value ? projectId.value : null);
         return Actions.sendResponse(res, 200, orgData);
       } catch (e) {
-        defaultLog.info('Error:', e);
+        defaultLog.error(e);
         return Actions.sendResponse(res, 400, e);
       }
     }
   } else {
+    defaultLog.error('Error getting project');
```

Comment:
again this could have more context with the error, otherwise not sure how helpful this message will be.